rotkehlchen/db/upgrades/v35_v36.py

```diff
@@ -616,6 +587,28 @@ def _fix_eth2_pnl_genesis(write_cursor: 'DBCursor') -> None:
         )
 
 
+def _reset_decoded_events(write_cursor: 'DBCursor') -> None:
+    write_cursor.execute('SELECT tx_hash from evm_transactions')
```

Comment:
when writting such a functions always add a docstring explaining what it does

Reply:
Can you also state that this needs to happen after `_upgrade_events_mappings` since is altering the schema